### PR TITLE
refactor getmovies to getLatestMovie for clarity

### PR DESCRIPTION
**Pull request by Jwisnewski**
Clarified definition of getLatestMovie in FetchMovies class.

### CODE REVIEW

File: spec/services/FetchMoviesSpec.php [hunk 1]
```diff
@@ -6,24 +6,23 @@
 
 describe(FetchMovies::class, function () {
 
-    describe("->getMovies()", function () {
+    describe("->getLatestMovie()", function () {
```
Comment (angelxmoreno) [inline spec/services/FetchMoviesSpec.php]: this test does not assert anything. at some point you need to call the new method

File: spec/services/FetchMoviesSpec.php [hunk 2]
```diff
@@ -6,24 +6,23 @@
...
             $fetcher = new FetchMovies();
             expect(FetchMovies::class)
             ->toReceive('getJsonString');
-            $fetcher->getMovies();
         });
 
         it('converts the json string into an array', function () {
             $fetcher = new FetchMovies();
-            $movieDataArr = $fetcher->getMovies();
+            $movieDataArr = $fetcher->getLatestMovie();
```
Comment (angelxmoreno) [inline spec/services/FetchMoviesSpec.php]: following the convention here, in the future, variables are underscored such that `$movieDataArr` should be `$movie_data_arr`. Nobiggie but start practicing.

File: spec/services/FetchMoviesSpec.php [hunk 3]
```diff
@@ -33,19 +32,16 @@
             expect($movieData->release_date)
             ->toBeA('string');
             expect($movieData->poster_image)
-            ->toBeA('null');
```
Comment (angelxmoreno) [inline spec/services/FetchMoviesSpec.php]: looks like here you are testing the movie data which is technically outside the scope of the testing the Fetcher

File: spec/services/FetchMoviesSpec.php [hunk 4]
```diff
@@ -33,19 +32,16 @@
...
         });
 
-        it('creates an array of Movie Models', function () {
-        });
-
         it("gets the latest movie", function () {
             $fetcher = new FetchMovies();
-            $fetcher->getMovies();
+            $fetcher->getLatestMovie();
             expect($fetcher)->toBeAnInstanceOf(FetchMovies::class);
```
Comment (angelxmoreno) [inline spec/services/FetchMoviesSpec.php]: here it looks like you are testing that php properly creates instances rather that what you said `gets the latest movie`

**Review by angelxmoreno — APPROVED** [review]
looks good. Minor observations

File: spec/services/FetchMoviesSpec.php (hunk 2 above)
Comment (Jwisnewski) [reply spec/services/FetchMoviesSpec.php]: cool should I refactor the var names in this scope or just continue in the future with this naming structure?

File: spec/services/FetchMoviesSpec.php (hunk 3 above)
Comment (Jwisnewski) [reply spec/services/FetchMoviesSpec.php]: Correct, I was initially testing to make sure it was getting the correct data in each property of the movie class instance. Instead I should just test the return of an instance of a movie class?

File: spec/services/FetchMoviesSpec.php (hunk 4 above)
Comment (Jwisnewski) [reply spec/services/FetchMoviesSpec.php]: So I should test for just the return of an instance of the movie class correct?

**Pull request merged**